packages/css/src/link/link.scss

```diff
@@ -6,45 +6,41 @@
 @import "../../utils/breakpoint";
 
 .amsterdam-link {
-  color: var(--amsterdam-link-color, blue);
+  color: var(--amsterdam-link-color);
   font-family: var(--amsterdam-paragraph-font-family);
   font-size: var(--amsterdam-link-font-size-narrow);
+  font-weight: var(--amsterdam-link-font-weight);
   line-height: var(--amsterdam-link-line-height);
   outline-offset: var(--amsterdam-link-outline-offset);
-  text-decoration: var(--amsterdam-link-text-decoration, underline);
-  text-decoration-color: var(--amsterdam-link-text-decoration-color, currentColor);
-  text-decoration-thickness: var(--amsterdam-link-text-decoration-thickness, 1px);
+  padding-block: 0.5rem;
```

Comment:
This seems dangerous as default

Reply:
You don't yet know where links are going to be used, the padding could become unwanted in certain use cases. Developers will create overrides.

Reply:
> Developers will create overrides.

They shouldn't ;) If the component height is based only on the `line-height` of its text, it's not large enough to comply with [this WCAG criterion](https://www.w3.org/WAI/WCAG21/Understanding/target-size.html). Same as the Button component for example, this component needs padding as part of its intrinsic height. Similar to Button, devs can mess with this padding, but that would mean they'd be 'breaking' the component

Reply:
Using `min-height` causes issues with the alignment of icons to text in links. For now we'll use padding.